test.sh

```diff
@@ -1,33 +1,12 @@
 #!/bin/sh
 # Running Parity Full Test Suite
+echo "________Running test.sh________"
 
 FEATURES="json-tests,ci-skip-issue"
 OPTIONS="--release"
 VALIDATE=1
 THREADS=8
 
-case $1 in
```

Comment:
this looks like no problem for the ci, but may break test on developers machines. maybe worth an announcement.

Reply:
removed it as never saw any parameters being passed to this script, but you're right, devs could make use of it.

Reply:
I remember, Pierre told it's not needed a while ago.